data/json/scenarios.json

```diff
@@ -380,6 +397,7 @@
     "id": "summer_advanced_start",
     "name": "The Next Summer",
     "points": 0,
+    "difficulty_impact": { "offence": "normal", "defence": "normal", "crafting": "normal", "environment": "normal", "social": "normal" },
```

Comment:
I would think the environment might be harder for any advanced time starts because of weather and evolution.  This is a really awesome first go at this kind of measurement.